packages/ubus-lime-utils/files/usr/libexec/rpcd/lime-utils-admin

```diff
@@ -106,6 +161,9 @@ if arg[1] == 'call' then
     elseif  arg[2] == 'firmware_upgrade' then firmware_upgrade(msg)
     elseif  arg[2] == 'last_upgrade_metadata' then last_upgrade_metadata(msg)
     elseif  arg[2] == 'firmware_confirm' then firmware_confirm(msg)
+    elseif  arg[2] == 'hotspot_wwan_enable' then hotspot_wwan_enable(msg)
```

Comment:
Needs to be hotspot_wwan_enable(unpack(msg)), and the same for hotspot_wwan_disable

Reply:
Thanks! That is why tests were needed :) I did not use unpack to maintain consistency.